integration-tests/integration_test.go

```diff
@@ -98,7 +101,7 @@ func TestProcfsScraperNoScrape(t *testing.T) {
 
 func TestProcfsScraperDisableFeatureFlag(t *testing.T) {
 	connScraperTestSuite := &suites.ProcfsScraperTestSuite{
-		TurnOffScrape:               false,
+		TurnOffScrape:               true,
```

Comment:
`DisableFeatureFlag` refers to `RoxProcessesListeningOnPort` being false. I don't think we should change the value here. I don't see a reason for it. You can make another test with TurnOffScrape=true and RoxProcessesListeningOnPort=false if you like, but this test should be kept. The name of the test could be changed to something like `ScrapeWithProcessesListeningOnPort`.

Reply:
Running this locally the test now fails with TurnOffScrape=false and RoxProcessesListeningOnPort=false. The reason is that previously if the process originator was nil the endpoint would also be nil, but now with the code changes, if the originator is nil the endpoint is not nil, so expecting that if RoxProcessesListeningOnPort=false that the endpoint is nil is not correct. This can be easily fixed and the test should pass.

Reply:
Fixed